lib/Mojo/mysql.pm

```diff
@@ -72,6 +72,10 @@ sub _dequeue {
   # you, silently, but only if certain env vars are set
   # hint: force-set mysql_auto_reconnect or whatever it's called to 0
   $dbh->{mysql_auto_reconnect} = 0;
+  # Maintain Commits with Mojo::mysql::Transaction
+  $dbh->{AutoCommit} = 1;
```

Comment:
Yes, it will override it. But setting AutoCommit = 0, needs explicit $dbh->commit, or $dbh->disconnect. We have Mojo::mysql::Transaction to deal with transactions.